### ISSUE TITLE
Zotero's Manuscript 'Type' is mapped to both biblatex's 'type' and 'howpublished'

### ISSUE THREAD

**Issue by chayaninw** [retorquere/zotero-better-bibtex#2114]
When I export a Manuscript item using Better BibLaTeX, the field `Type` in Zotero is simultaneously mapped to both `type` and `howpublished` in the `.bib` file.

This causes the content of `Type` to be printed twice in some bibliography styles, including the default `biblatex` styles.

support log ID: YEJ5PWNR-refs-euc

**Comment by chayaninw** [comment]
Yes, now it only exports to `type`. Thanks for the prompt response and the for this great add-on!

I apologise for not having done this when I first submitted the issue, but I did a bit of digging on whether `type` or `howpublished` is expected here and it seems like `howpublished` might be the better choice, at least if I understand the meaning of Zotero's `Type` field correctly.

The documentation of `biblatex` describes the `unpublished` entry type as
> A work with an author and a title which has not been formally published, such as a manuscript or the script of a talk. Use the fields `howpublished` and `note` to supply additional information in free format, if applicable.
However, both `howpublished` and `type` are supported optional fields.

With the default `biblatex` styles it doesn't matter much, since it simply prints one after another in the bibliography entry. However, some other styles treat them differently and it seems they expect `howpublished` as suggested in the `biblatex` documentation.

* `biblatex-chicago`: Only `howpublished` is printed, `type` is ignored
* `biblatex-oxref`: Only `howpublished` is printed, `type` is ignored
* `biblatex-apa`: Both are printed, but differently. Checking with the APA manual, it seems `howpublished` would give the correct result.
